docs/en/latest/admin-api.md

```diff
@@ -547,11 +547,11 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 `hash_on` can be set to different types:
 
 1. when it is `vars`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html), without the prefix '$'.
-1. when it is `header`, the `key` is required. It is equal to "http_`key`".
```

Comment:
We don't need to change this. Markdown will handle the order automatically. There is no need to maintain the order by us.

Reply:
updated.